src/binding/fortran/mpif_h/buildiface

```diff
@@ -125,7 +126,8 @@ foreach $_ (@ARGV) {
     if (/-noprototypes/) { $build_prototypes = 0; }
     elsif (/-infile=(.*)/) {
 	# Special arg to help with debugging
-	$prototype_file = $1;
+        $prototype_file_a = $1;
```

Comment:
weird whitespace

Reply:
This is due to old scripts had `TAB`s. Added a commit to replace all TABs and trailing spaces.